Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -239,6 +244,22 @@ void ABenchmarkGymGameModeBase::Tick(float DeltaSeconds)
 	{
 		PrintMetricsTimer.SetTimer(10);
 	}
+	
+	if (StatStartFileTimer.HasTimerGoneOff())
+	{
+		USpatialNetDriver* SpatialDriver = Cast<USpatialNetDriver>(GetNetDriver());
+		if (ensure(SpatialDriver != nullptr))
+		{
+			FString FileName = FString::Format(TEXT("stat startfile {0}.ue4stats"), { SpatialDriver->Connection->GetWorkerId() });
+			GEngine->Exec(GetWorld(), *FileName);
+			StatStartFileTimer.SetTimer(999999);
```

Comment:
```suggestion
			StatStartFileTimer.SetTimer(999999);
			StatStopFileTimer.SetTimer(60);
```
We could trigger the stat collection a while after deployment start